src/music/music.repository.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable } from '@nestjs/common';
+import { CreateMusicDto } from './dto/create-music.dto';
+import { UpdateMusicDto } from './dto/update-music.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Music } from './entities/music.entity';
+import { Repository } from 'typeorm';
+
+@Injectable()
+export class MusicRepository {
+    constructor(@InjectRepository (Music)
+private MusicRepository: Repository <Music>) {}
+  async create(createMusicDto: CreateMusicDto) {
+    
+    const music =  await this.MusicRepository
+    .createQueryBuilder()
+    .insert()
+    .into(Music)
+    .values(createMusicDto)
+    .execute()
+
+    return music.generatedMaps[0]
+    
+  }
+
+  findAll() {
+    return this.MusicRepository
+    .createQueryBuilder()
+    .getMany()
+  }
+
+  findOne(id: number) {
```

Comment:
same change everywhere to async functions

Reply:
Done